src/cogs/redditAPI.py

```diff
@@ -56,6 +67,131 @@ async def memes(self, ctx, arg : int, x = 1):
             em2 = discord.Embed(title = "Range Exceeded", description = "We are bound to provide you maximum 5 memes at a time", color = discord.Color.green())
             await ctx.send(embed = em2)
 
+    # Command for starting AutoPost
+    @commands.command(aliases = ['apon'])
+    async def autoposton(self, ctx, channel : commands.TextChannelConverter):
+        with open ('./cogs/autoid.json', 'r') as f:
+            ap_channels = json.load(f)
+
+        ap_channels[str(ctx.guild.id)] = str(channel.id)
+
+        with open ('./cogs/autoid.json', 'w') as f:
+            json.dump(ap_channels, f, indent = 4)
+        ap_channel_list.append(channel.id)
+
+    # Command for ending AutoPost
+    @commands.command(aliases = ['apoff'])
+    async def autopostoff(self, ctx):
+        with open ('./cogs/autoid.json', 'r' ) as f:
+            ap_channels = json.load(f)
+        del_channel = int(ap_channels[str(ctx.guild.id)])
+        ap_channels[str(ctx.guild.id)] = "0"
+
+        with open ('./cogs/autoid.json', 'w') as f:
+            json.dump(ap_channels, f, indent = 4)
+        ap_channel_list.remove(del_channel)
+
+
+    # Loop for autoposting every 15 minutes
+    @tasks.loop(minutes = 15)
+    async def test(self):
+        
+        reddit = asyncpraw.Reddit(client_id = "Vng40QEkdlR_VuVphsbxxA", client_secret = "4evSCjg47N9CTtQqqyQUuA8X2I4qXQ", username = "ProfessionalFloor135",  password =  "Qwerty@18606", user_agent = 'EduMemebot')
```

Comment:
Is it okay to provide the client credentials?

Reply:
it is, but i think it's better if another alternative is worked around.

Reply:
@Rajrup911 try putting these credentials in a json file and place it in .gitignore